app/templates/Email/nearEventsEmail.latte

```diff
@@ -1,19 +1,13 @@
-<p>Ahoj hvězdo! </p>
+<p>Ahoj,</p>
 
-<p>Blíží se nám tu nějaké akce a ty jsi se zatím nerozhodl, jestli na ně pojedeš. Buď potvrď účast a my budeme skákat do stropu
-    radostí, nebo je normálně odmítni. Aspoň tě pak nebudeme otravovat.</p>
+<p>Následující akce jsou už za rohem a zatím nemáme tvé vyjádření, jestli pojedeš. </p>
```

Comment:
Dal bych spíš "**n**ásledující akce jsou už za rohem**, ale** zatím **jsi ještě nedal vědět, jestli pojedeš. Potvrď prosím, jestli se zúčastníš, nebo nemůžeš.**"

Reply:
Ještě to "N" v "Následující" bych změnil na malé, není asi gramaticky nutnost, ale přijde mi to hezčí.